examples/git_history/index.html

```diff
@@ -14,20 +14,28 @@
 
     <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1, minimum-scale=1, user-scalable=no">
 
-    <script src="mobile_fix.js"></script>
     <script src="perspective.view.js"></script>

```

Comment:
I think all of these can be refactored to be from `node_modules/@jpmorganchase/perspective`.

Reply:
This is specific to formatting `<perspective-viewer>` to not scroll the viewport on mobile because these demos are full-page - I believe this is specific only to these examples' usage of `<perspective-viewer>`.

However, we don't host these sites anymore so no need for a mobile_fix at all - will delete this.

Reply:
Actually scratch that comment - this commit is outdated, this was already deleted.